src/ResourceManager/Sql/Commands.Sql/Common/AzureSqlCmdletBase.cs

```diff
@@ -59,87 +36,5 @@ protected virtual string GetResourceId(M model)
             HelpMessage = "The name of the resource group.")]
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
```

Comment:
@shahabhijeet is relaxing the requirements of parameters a breaking change?  I'm doing this change because there have been multiple github issues requesting it. eg: #635